tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -116,6 +111,16 @@ static void execute(String... args) throws Exception {
         }
     }
 
+    private static Properties getProperties(File optionalCommandConfig) throws TerseException, IOException {
+        if (optionalCommandConfig == null) {
+            return new Properties();
+        } else {
+            if (!optionalCommandConfig.exists())
+                throw new TerseException("Properties file " + optionalCommandConfig.getPath() + " does not exists!");
```

Comment:
Minor but I would add a `{ }` block statement instead of using single line statement for `if`.

Reply:
That's not required by the coding guidelines though.